tutor/src/components/book-page.js

```diff
@@ -24,14 +24,33 @@ const LEARNING_OBJECTIVE_SELECTORS = '.learning-objectives, [data-type=abstract]
 const IS_INTRO_SELECTORS = '.splash img, [data-type="cnx.flag.introduction"]';
 const INTER_BOOK_LINKS = 'a[href^=\'/book/\']';
 
+
+function hasSiblingFigure(el) {
+  return Boolean(
+    (el.previousElementSibling &&
+      dom(el.previousElementSibling).matches('.os-figure')
+    ) || (
+      el.nextElementSibling && dom(el.nextElementSibling).matches('.os-figure')
+    )
+  );
+}
+
 // called with the context set to the image
 function processImage() {
   const img = dom(this);
   const figure = img.closest('.os-figure')
     || img.closest('figure')
     || img.closest('[data-type=media]');
   if (!figure) { return; }
+
+  if (figure.querySelector('.splash')) {
+    figure.classList.add('full-width');
+  }
   if (figure.classList.contains('splash')) { return; }
+  // figures that are not in a series
+  if (!hasSiblingFigure(figure)) {
+    figure.classList.add('standalone');
```

Comment:
Didn't you already do this on another PR but called this 'independent'?

Reply:
Ah nvm I see you just renamed it

Reply:
yeah, i thought "standalone" was clearer but 🤷‍♂